common/flatpak-dir.c

```diff
@@ -6381,6 +6381,11 @@ flatpak_dir_update (FlatpakDir          *self,
   if (deploy_data != NULL)
     old_subpaths = flatpak_deploy_data_get_subpaths (deploy_data);
 
+  /* If the existing pull is partial, disable static deltas. They can
+     break, because ostree assumes all old objects are locally available. */
```

Comment:
Hmm...but we check if commits are partial before using deltas.  See https://github.com/ostreedev/ostree/blob/master/src/libostree/ostree-repo-pull.c#L3125 etc.

Reply:
Really, that is weird. I wonder what happened then?

Reply:
Oh, its probably not picking up commitpartial state from the parent repo